app/assets/stylesheets/spinner.scss

```diff
@@ -0,0 +1,13 @@
+@keyframes crayons-spinner {
+  0% {
+    transform: rotate(0deg);
+  }
+  100% {
+    transform: rotate(360deg);
+  }
+}
+
+.crayons-spinner {
+  color: var(--accent-brand);
+  animation: 0.75s linear infinite crayons-spinner;
+}
```

Comment:
One last thing - should we have this file in `/stylesheets/components` folder?

Reply:
I was thinking about this too. I currently have it as an app component in Storybook, but should this icon be considered as part of the design system instead of an app component? I'm thinking yes.